### PR TITLE
Added 2017 Impactkit summary

### PR DESCRIPTION
**Pull request by lcapriot**

### CODE REVIEW

File: _posts/2017-05-03-impactkit.md [hunk 1]
```diff
@@ -6,10 +6,108 @@ categories: archive impactkit
 indico: https://indico.cern.ch/event/624551/
 ---
 
-The Impact Kit is a three-day course in which participants follow advanced
-lectures and get a chance to obtain some hands-on experience on the LHCb
-software. Projects are proposed, and in addition participants are encouraged to
-bring their own problems to the table.
+The second LHCb Impactkit workshop was held at CERN from 3rd May and 5th May 2017, with 19 participants from 13 institutions across 5 countries.
```
Comment (vbellee) [inline _posts/2017-05-03-impactkit.md]: from 3rd May and 5th May 2017 -> from May 3 to May 5, 2017

File: _posts/2017-05-03-impactkit.md [hunk 2]
```diff
@@ -6,10 +6,108 @@ categories: archive impactkit
 indico: https://indico.cern.ch/event/624551/
 ---
 
-The Impact Kit is a three-day course in which participants follow advanced
-lectures and get a chance to obtain some hands-on experience on the LHCb
-software. Projects are proposed, and in addition participants are encouraged to
-bring their own problems to the table.
+The second LHCb Impactkit workshop was held at CERN from 3rd May and 5th May 2017, with 19 participants from 13 institutions across 5 countries. 
+The students were taught advanced DaVinci algorithms, HTL TisTos and Turbo, how to
```
Comment (vbellee) [inline _posts/2017-05-03-impactkit.md]: HTL -> Hlt

File: _posts/2017-05-03-impactkit.md [hunk 3]
```diff
@@ -6,10 +6,108 @@ categories: archive impactkit
...
+
+Here are some photos taken on the last day:
+
+![Heavy chopping]({{ site.baseurl }}/img/2017-05/foto0.jpg)
+![Puttanesca sauce in all its beauty]({{ site.baseurl }}/img/2017-05/foto1.jpg)
+![Hackathon in progress]({{ site.baseurl }}/img/2017-05/foto2.jpg)
+![Dinner at the pit]({{ site.baseurl }}/img/2017-05/foto3.jpg)
+
+
+The Starterkit team would like to thank all the students for attending. Our passion in teaching would serve no purpose without your will to learn.
```
Comment (vbellee) [inline _posts/2017-05-03-impactkit.md]: passion in -> passion for

File: _posts/2017-05-03-impactkit.md (hunk 1 above)
Comment (lcapriot) [reply _posts/2017-05-03-impactkit.md]: I partly disagree, and changed it to:
from 3rd to 5th May 2017
Waiting for a native speaker to comment!

File: _posts/2017-05-03-impactkit.md (hunk 2 above)
Comment (lcapriot) [reply _posts/2017-05-03-impactkit.md]: Done

File: _posts/2017-05-03-impactkit.md (hunk 3 above)
Comment (lcapriot) [reply _posts/2017-05-03-impactkit.md]: Done

File: _posts/2017-05-03-impactkit.md [hunk 4]
```diff
@@ -6,10 +6,108 @@ categories: archive impactkit
 indico: https://indico.cern.ch/event/624551/
 ---
 
-The Impact Kit is a three-day course in which participants follow advanced
-lectures and get a chance to obtain some hands-on experience on the LHCb
-software. Projects are proposed, and in addition participants are encouraged to
-bring their own problems to the table.
+The second LHCb Impactkit workshop was held at CERN from 3rd May and 5th May 2017, with 19 participants from 13 institutions across 5 countries. 
+The students were taught advanced DaVinci algorithms, HTL TisTos and Turbo, how to 
+write a DecFile, how to manage data on the grid with Ganga and how to develop LHCb software using LHCb git. It was an intense couple of days which ended in a 1.5 days long hackathon, where the students, divided in groups of 2, had to work on projects regarding mainly collaboration-wide issues or on other personal projects in line with the subjects of the lectures.
```
Comment (vbellee) [inline _posts/2017-05-03-impactkit.md]: Maybe give an idea of some of the projects chosen.

**Pull request merged**